start-demo.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -e
+
+#
+# Handling the case of running directly from the Maven package root folder versus directly from the source code
+#
+
+if [ "${true}" == "" ]; then
+    # Running from repo
+    version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+    artifactId=$(mvn help:evaluate -Dexpression=project.artifactId -q -DforceStdout)  
+    buildDir=target/$artifactId-$version/
+    echo "[INFO] Running from repo root directory. Moving to the build dir: '$buildDir'..."
+    cd $buildDir
+fi
+
+# Override the setup-dirs.sh file:
+cat > run/docker/scripts/setup-dirs.sh << EOL
+#!/usr/bin/env bash
+set -e
```

Comment:
@rbuisson  are we missing `./fetch-ozone-distro.sh 1.0.0-SNAPSHOT` here?

Reply:
Not exactly, as the distro is expected to be built and therefore present locally (in the target/ directory).
However, this setup-dirs.sh file is deprecated from my recent changes in Ozone Docker. Let me fix that.